fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/ResourceManagerHelperImpl.java

```diff
@@ -65,10 +65,9 @@ public static ResourceManagerHelperImpl get(ResourceType type) {
 	 * @return {@code true} if successfully registered the resource pack, else {@code false}
 	 * @see ResourceManagerHelper#registerBuiltinResourcePack(Identifier, ModContainer, String, ResourcePackActivationType)
 	 * @see ResourceManagerHelper#registerBuiltinResourcePack(Identifier, ModContainer, ResourcePackActivationType)
-	 * @see ResourceManagerHelper#registerBuiltinResourcePack(Identifier, String, ModContainer, boolean)
 	 */
 	public static boolean registerBuiltinResourcePack(Identifier id, String subPath, ModContainer container, String displayName, ResourcePackActivationType activationType) {
-		String separator = container.getRootPath().getFileSystem().getSeparator();
+		String separator = container.getRootPaths().get(0).getFileSystem().getSeparator();
```

Comment:
Is this algorithm right? i.e. is it possible for builtin resource packs to be stored in non-first root path?

Reply:
It is, but this is only for the FS separator. Packs split across multiple root paths should be handled correctly (in the resource pack impl).

Reply:
Yeah, all filesystems should have the same file separator, so doesn't matter.